Create 2021-05-02-UNCC-hierbin.csv

lunarmouse opened the pull request:
## Description

Please include a brief description of your change.  

---
> _If this section does not apply to you, please delete it in your PR_  

If this is a **new team submission**, please include the following details :-  
- Team name: UNCC
- Model Name: hierbin
- Institution:  UNC Charlotte Department of Public Health Sciences, UNC Charlotte School of Data Science
---

> _If this section does not apply to you, please delete it in your PR_  

If you are **adding new scenarios** to an existing model, please include the following details:- 
- Team name: 
- Model name that is being updated:  
---

## Checklist

- [x] Specify a proper PR title with your team name.
- [ ] All validation checks ran successfully on your branch. Instructions to run the tests locally is present [here](https://github.com/midas-network/covid19-scenario-modeling-hub/wiki/Running-Checks-Locally).

lunarmouse commented:
[Uploading 2021-05-02-UNCC-hierbin.csv…]()

LucieContamin commented:
Hi @lunarmouse ,

Thank you for your pull-request. However, the csv file joined in the pull-request seems to be empty. Would it be possible to upload it again? 
Also, would it be possible to add a metadata file with your submission? You can find information here: https://github.com/midas-network/covid19-scenario-modeling-hub/blob/master/data-processed/METADATA.md. For additional information, we will accept a file without the fields that have "Not applicable" as possible values. By default, we will confider all these missing fields with the value "Not applicable".

Please, feel free to let me know if you have any questions or if I can be of further assistance.

Thanks,
Lucie

lunarmouse commented:
Hi Lucie,
Thanks very much for your feedback. I tried again this time and it seems
the zipped file should work now. I will create the metadata file soon.

Best,
Shi

On Wed, May 5, 2021 at 9:02 AM Lucie Contamin ***@***.***>
wrote:

> [*Caution*: Email from External Sender. Do not click or open links or
> attachments unless you know this sender.]
>
>
> Hi @lunarmouse <https://github.com/lunarmouse> ,
>
> Thank you for your pull-request. However, the csv file joined in the
> pull-request seems to be empty. Would it be possible to upload it again?
> Also, would it be possible to add a metadata file with your submission?
> You can find information here:
> https://github.com/midas-network/covid19-scenario-modeling-hub/blob/master/data-processed/METADATA.md.
> For additional information, we will accept a file without the fields that
> have "Not applicable" as possible values. By default, we will confider all
> these missing fields with the value "Not applicable".
>
> Please, feel free to let me know if you have any questions or if I can be
> of further assistance.
>
> Thanks,
> Lucie
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/midas-network/covid19-scenario-modeling-hub/pull/51#issuecomment-832668415>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH3EIASQ4GIJ7RC6QSAPS7DTME6WLANCNFSM44DYXDIQ>
> .
>


-- 
Shi Chen, Ph.D.
Assistant Professor of Health Informatics and Analytics
Department of Public Health Sciences
College of Health and Human Services
School of Data Science
University of North Carolina Charlotte

LucieContamin commented:
Hi Shi,

Thank you for the update!

I have some questions about the ZIP file. It seems that some information are missing. 
The `location` "US" for national level scenarios and the `type` "point" are both missing. Would it be possible to add it, please? 
Also, I notice that the `target_end_date` date value are all for Sunday. However, the `target_end_date` format should be the Saturday at the end of the week time period. Would it also be possible to update it, please?
You can find more information about the data submissions format here: https://github.com/midas-network/covid19-scenario-modeling-hub/tree/master/data-processed#data-submission-instructions

Thanks for joining the metadata file. Just for information, I will remove the line breaks between each section to comply with the metadata format after merging the pull-request.

Please don't hesitate to contact me if you have any questions or if I can be of further assistance.

Thanks,
Lucie

lunarmouse commented:
Hi Lucie thanks very much for checking the file for me. I have added the US for national level scenario and corrected the target end date to reflect the epi-week (i.e., changing from. Sunday to Saturday, sorry for the mistake). 
However it is a bit challenge to provide point estimate -- the output file is quite complicated and huge. I am a bit worried that mixing character (e.g. "NA") with actual quantile would cause some problems in the file. I did a quick test in the result file and it seems that most of the results are quite normally distributed, so if possible could you please use 50 percentile (median) for visualization purpose?

csmith701 commented:
Hi Shi,

I took a look at the most recent upload this morning and noticed a few other formatting issues. The dates need to be formatted at "yyyy-mm-dd" both for the `target_end_date` and `model_projection_date` columns. Additionally, in the `location` column it appears that the leading zero has been dropped from the location codes (e.g. needs to be "01" not "1"). Finally, given the way our plotting code is set up, we need teams to provide a point estimate (even if it is just the median). Would it be possible for you to add rows for the point estimates to your submission?

Thanks,
Claire

lunarmouse commented:
Hi Claire, thanks for your feedback. I have made the requested change to the newest file.

LucieContamin commented:
Hi Shi, 

Thank you for all the updates. 

I have another question, the `location` column contains a value `43`. However, I can not link it to a states names. Would it be possible to give me more information on that, please? You can also find the information we use here: https://github.com/midas-network/covid19-scenario-modeling-hub/blob/master/data-locations/locations.csv 

Also, just for information, I will rename the ZIP file when I merge it to `2021-05-02-UNCC-hierbin` to follow the name convention and I will remove the row names in the file.

Please don't hesitate to contact me if you have any questions or if I can be of further assistance.

Thanks,
Lucie

csmith701 commented:
Hi Shi,

Just some follow up info on Lucie's comment. It looks like FIPS code 43 used to be used for Puerto Rico but has been retired. 72 is now the code used for Puerto Rico.

Best,
Claire
